engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -867,7 +867,12 @@ object Main {
             )
           val res = main.execute(parsedArgs: _*)
           if (!res.isNull) {
-            out.println(res);
+            val textRes = if (res.isString) {
+              res.asString
+            } else {
+              res.toString
+            }
+            out.println(textRes);
```

Comment:
Out of curiosity - what does this change?

Reply:
1152e3d21edc84fcd086a62fc9970dfe8d0da7a0

Reply:
Without that change the multiline output of https://github.com/enso-org/enso/pull/8608#issuecomment-1867346712 wouldn't be that nice.